### PR TITLE
[WIP] Create category shortcut in experimental product page

### PR DESCRIPTION
**Pull request by zuk3975**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | TBD
| Type?             | refacto
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | part of https://github.com/PrestaShop/PrestaShop/issues/10632
| How to test?      | TBD
| Possible impacts? | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25083)

### CODE REVIEW

File: admin-dev/themes/new-theme/js/pages/product/product-map.js [hunk 1]
```diff
@@ -24,6 +24,7 @@
  */
 
 const combinationListId = '#combination_list';
+const productCategoriesFormId = '#product_categories';
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/js/pages/product/product-map.js:27]:
```suggestion
```

File: src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php [hunk 2]
```diff
@@ -144,6 +144,20 @@ public function searchGridAction(Request $request)
         );
     }
 
+    public function quickCreateAction(Request $request): JsonResponse
```
Comment (Progi1984) [inline src/PrestaShopBundle/Controller/Admin/Sell/Catalog/CategoryController.php:147]: Please add PHPDoc

**Comment by zuk3975** [comment]
blocked by https://github.com/PrestaShop/PrestaShop/pull/24878 (modified same js file, avoiding conflicts)

**Review by Progi1984 — COMMENTED** [review]
Some feedbacks

**Comment by Julievrz** [comment]
> This pull request seems to contain new translation strings. I have summarized them below to ease up review:
> 
> * `Admin.Catalog.Feature`
>   
>   * [ ]  [`Create a category`](https://github.com/PrestaShop/PrestaShop/pull/25083/files#diff-cf58466900f44e9e8c7c065bbb2021825ddc63ed954aed39a7b6c560a68267efR49)
>   * [ ]  [`New category name`](https://github.com/PrestaShop/PrestaShop/pull/25083/files#diff-fce8961015f8ff335e741b2353b9e83dcdc9a843a92a4874a0bcd3be3fb133ceR57)
> * `Admin.Catalog.Help`
>   
>   * [ ]  [`Search categories`](https://github.com/PrestaShop/PrestaShop/pull/25083/files#diff-fce8961015f8ff335e741b2353b9e83dcdc9a843a92a4874a0bcd3be3fb133ceR73)
> 
> (Note: this is an automated message, but answering it will reach a real human)

These are no new wordings. 👀

**Review by Julievrz — APPROVED** [review]
Wording ✅

**Comment by Progi1984** [comment]
@zuk3975 Could you rebase your PR ?

**Comment by zuk3975** [comment]
The feature specification changed and I started new PR (https://github.com/PrestaShop/PrestaShop/pull/25342). So this can be closed.

**Pull request closed without merging**